sdk/bootstrap/main.go

```diff
@@ -305,6 +314,51 @@ func installDCOSCertIntoJRE() {
 	log.Println("Successfully installed the certificate.")
 }
 
+func decodeKeystores() error {
```

Comment:
There are currently no tests for bootstrap utility. I can add `decode` tests and add test evaluation to the build pipeline.

There is also gradle plugin that we could use to build this package instead of custom bash script -  https://github.com/echocat/gradle-golang-plugin .